modules/authentication/proc-enabling-authentication-with-github.adoc

```diff
@@ -158,6 +159,7 @@ auth:
       production:
         enterpriseInstanceUrl: ${GITHUB_HOST_DOMAIN}
 ----
+////
```

Comment:
```suggestion
//// RHIDP-5765 - Comment out references to GH enterprise
```

Reply:
I believe this is not what you intend to do. `////` are delimiters for is a multi-line comment. This text would be outside the comment and be published. https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#comments

Additionally, this is duplicating an already available information. The Jira reference will appear as part of the git history.